dra-evolution/testdata/pod-one-container-one-gpu-one-vf.yaml

```diff
@@ -5,20 +5,63 @@
 # one, since netdev doesn't get added to any container, it would
 # just be ignored for that purpose.
 
+apiVersion: resource.k8s.io/v1alpha2
+kind: ResourceClaimTemplate
+metadata:
+  name: template-foozer-plus-vf
+spec:
+  metadata:
+    labels:
+      foo: bar
+  spec:
+    constraints:
+    - match:
+        attribute: pcie-root.dra.k8s.io
+    requests:
+    - deviceClassName: example.com-foozer
+    - deviceClassName: example.org-sriov-nic
+      requirements:
+      - device:
+          selector: "device.stringAttributes['sriovType.example.org-sriov-nic'] == 'vf'"
+---
+apiVersion: resource.k8s.io/v1alpha2
+kind: Pod
+metadata:
+  name: foozer
+  namespace: default
+  uid: abcd-1234
+spec:
+  containers:
+  - image: registry.k8s.io/pause:3.6
+    name: my-container
+    resources:
+      requests:
+        cpu: 10m
+        memory: 10Mi
+      claims:
+      - name: gpu-and-nic
```

Comment:
So, nics are not mounted in containers, but in the pod. Should this be a two-level mount key instead:

```
claims:
- name: gpu-and-nic
  request: gpu
```

(probably for nics we can just *ignore* this mount, but the point is suppose we had some other device in there but we wanted it split across containers...)

Reply:
I believe this already exists in the dra-evolution prototype (unless it got cut in a recent revision). I know I used this in some of the examples I put together.

Reply:
See also https://github.com/kubernetes-sigs/wg-device-management/pull/14/files#r1615942525